### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 9

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#491] [Esami & Esoneri M-Z]
### Traccia

<img width="553" alt="image" src="https://github.com/user-attachments/assets/3d3df002-42e6-4e14-85b9-18e398c1be4e">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/ebe011a4-f48e-44fc-a8ba-2ebac6cc2997)

## Tableau

![image](https://github.com/user-attachments/assets/c5ee40f2-ceb8-4263-a9c0-169674aa7795)

![image](https://github.com/user-attachments/assets/de9ad42e-2b3a-4f82-9d26-a39c5f517282)
